src/borg/constants.py

```diff
@@ -123,6 +123,15 @@ class KeyType:
     BLAKE2REPO = 0x05
     BLAKE2AUTHENTICATED = 0x06
     AUTHENTICATED = 0x07
+    # new crypto starts at 0xa0
+    AESOCBKEYFILE = 0xa0
+    AESOCBREPO = 0xa1
+    CHPOKEYFILE = 0xa2
+    CHPOREPO = 0xa3
+    BLAKE2AESOCBKEYFILE = 0xa4
+    BLAKE2AESOCBREPO = 0xa5
+    BLAKE2CHPOKEYFILE = 0xa6
+    BLAKE2CHPOREPO = 0xa7
```

Comment:
I dislike how this grows like `n * m * o` (and I did not yet implement blake2b id hashing here, for that i would've needed 8 keytypes already).
n == amount of key storage places (currently 2)
m == amount of id hashing algorithms (currently 2, hmac-sha256 and blake2b)
o == amount of aead ciphersuites (now 3, aes-ctr-hmac-sha256 and 2 new aes-ocb and chacha20-poly1305)

in the future, guess m could grow due to blake3.